src/xcode/ENA/ENA/Source/Scenes/UpdateOS/UpdateOSViewModel.swift

```diff
@@ -0,0 +1,14 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+
+struct UpdateOSViewModel {
+	// swiftlint:disable:next force_unwrapping
+	let image = UIImage(named: "Illu_UpdateOS")!
+	// swiftlint:disable:next force_unwrapping
```

Comment:
```suggestion
	let image = UIImage(imageLiteralResourceName:: "Illu_UpdateOS")
```

Reply:
Today I learned :D
Wow that API exists since iOS 2.0 :o 
Thanks!